portfolio/src/main/webapp/index.html

```diff
@@ -12,23 +12,153 @@
+                    some popular flash games and pasted them into my site. Popular sites like Addicting Games and Candystand were blocked at my school but my domain wasn't, so I took game requests from my friends 
+                    and the site became pretty popular. I quickly applied for Google Adsense to try and get some revenue, and told my friends to click to ads to get me some money, but I should've known
+                    Google would catch on and my account would be suspended, but at this point I didn't care about the money...I loved building something people used. I tried to add
+                    other features like posting and social networking, but the drag-and-drop website builder that I used didn't support that feature. After some searching
+                    I figured out that I needed to code it. I then googled "How does Facebook code their website" and figured out they used PHP, then searched "How to create posts in PHP". 
+                    I watched Youtube videos for days and copied the methods I saw, tweaking them for my own site, and eventually created my first social network: 
+                    <a href="http://2wocents.com" style="font-weight: bold;">2wocents.com</a>. While doing this, I noticed that I could code for 8+ hours in one sitting, and figured out that this
+                    is what I loved to do.
+                </p>
+
+                <p id="bio5" class="bios"> All in all, I feel I am where I am today because of the blessings that I've received, the people who have sacrificed to get me here, and the work that I had to do
+                    with the support of my loved ones. While I've come a long way, the exciting thing is that I'm nowhere close to where I want to be.
+                </p>
+
+                <p id="bio6" class="bios"> This is who I am. 
+                </p>
 
-    <div id="content">
-      <h1>Start</h1>
-      <p>This is my portfolio.</p>
-      <p>My name is Seun aka computabeast.</p>
-      <img class="contentimages" src="images/headshot.jpg"/>
+                <img src="images/signature2.png" style="height: 50px; width: auto;" />
+            
+            </div>
+            <div id="horizontal-spacer"></div>
+            <div id="pictureTable">
+                <table id="tabler">
+
+                    <tr>
+                        <td><img class="tablepics rightpics" src="images/coolpic.jpg"</td>
+                        <td><img class="tablepics rightpics" src="images/jogging.jpg"></td>
+                    </tr>
+
+                    <tr>
+                        <td><img class="tablepics rightpics" src="images/squad.jpg"></td>
+                        <td></td>
+                    </tr>
+                </table>
+            </div>
+        </div>
+
+
+        <div class="part">
+            <div id="links-table">
+                <table id="tabler">
+
```

Comment:
nit: remove the extra line here, since it's not really adding to the readability of this section and there isn't an extra line above the closing table tag